src/en/credentials.md

```diff
@@ -262,9 +284,10 @@ local-credentials:
 Notice how the output says 'local-credentials', meaning they are stored on
 the local Juju client.
 
-#### In use
+### Listing remote credentials
 
-To see what credentials are in use by a model (here the 'default' model):
+To see what credentials are in use by a model (here the 'default' model) the
```

Comment:
Plural? not singular? 'what credential is in use'? A model can only have one credential...

Reply:
Nicely seen. Will reword. But this intrigues me. If a model is shared with another user isn't it reasonable to expect that user to want to add their own backing cloud authentication material (e.g. their AWS account) to use it? Why the limitation of one credential per model?

Reply:
Credential per model. This means that all communications with the cloud that this model does is linked to one credential, and potentially, is billed to one account.

If a user wants to use their own credential, they can either create another model specifying --credential; or now use 'set-credential'.

Eventually a credential can be shared between users. But this is another story and a bit further in the future.